Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -97,8 +97,29 @@ def get_config ():
       'ecptr'     :   'EC/ecregionpointer.bin',
       'gbe'       :   'GBE/gbe.bin',
       'bpm'       :   'bpmgen2',
+      'bpm_params':   'bpmgen2/Example.bpDef',
```

Comment:
Where user could get bpmgen2/Example.bpDef?

Reply:
It is from BPMGen2 package. they get it when they download BpmGen2 package

Reply:
Got it.